webxdc.d.ts

```diff
@@ -74,19 +93,25 @@ interface Webxdc<T> {
    * @returns promise that resolves when the listener has processed all the update messages known at the time when `setUpdateListener` was called.
    * */
   setUpdateListener(
-    cb: (statusUpdate: ReceivedStatusUpdate<T>) => void,
+    cb: (statusUpdate: ReceivedStatusUpdate<StatusPayload>) => void,
     serial?: number
   ): Promise<void>;
+
+  /**
+   * Join a realtime channel.
+   */
+  joinRealtimeChannel(): RealtimeListener;
```

Comment:
does this need to say it can also return `null` (if you call it a second time)?

Reply:
This should probably even return an error to make misusage more clear

Reply:
IMO wither specify in the jsdoc comment that it throws an error or specify `| null` as return type and also document it in the jsdoc comment